src/pages/manage_dac/ManageDac.js

```diff
@@ -58,7 +64,29 @@ class ManageDac extends Component {
         });
       }
     );
-  }
+  };
+
+  getUserRole = async () => {
+    let roles;
+    let dacIDs = [];
+    let res;
+    await User.getMe().then(
```

Comment:
You don't need to do this here. When a user logs in, we store their roles as part of their stored user object. See https://github.com/DataBiosphere/duos-ui/blob/develop/src/libs/storage.js for where we do this.

Reply:
so, maybe you will know why this is happening, but I tried to use the Storage.getCurrentUserRole() method originally, but combined with the .then() I get this error: 
![image](https://user-images.githubusercontent.com/77015255/111525177-ad916180-8733-11eb-8bd0-5e5b6755b9f9.png)
So that is why I switched over to the User.getMe() call, and then just use result.roles instead of result

Reply:
This code will do what you're looking to do without the /me call and doesn't trigger the error you see:
```
    const currentUser = Storage.getCurrentUser();
    let dacIDs = [];
    currentUser.roles.forEach((role) => {
      if (role.name === CHAIR) {
        dacIDs.push(role.dacId);
      }
    });
```

Reply:
That being said, the look can be cleaned up by filtering the roles on name, then mapping to return a list of dacIds.

Reply:
good idea!!!